src/components/Page-Title/Page-Title.js

```diff
@@ -0,0 +1,64 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+export default class PageTitle extends React.Component {
+  render() {
+    let help = "";
```

Comment:
```suggestion
    let help = '';
```
The linter complains about double quotes, use single quotes instead, please.